e2e/profile.spec.ts

```diff
@@ -0,0 +1,159 @@
+import { test, expect } from '@playwright/test';
+
+const certs = [
+  {
+    name: 'Responsive Web Design',
+    url: '/certification/certifieduser/responsive-web-design'
+  },
+  {
+    name: 'JavaScript Algorithms and Data Structures',
+    url: '/certification/certifieduser/javascript-algorithms-and-data-structures'
+  },
+  {
+    name: 'Front End Development Libraries',
+    url: '/certification/certifieduser/front-end-development-libraries'
+  },
+  {
+    name: 'Data Visualization',
+    url: '/certification/certifieduser/data-visualization'
+  },
+  {
+    name: 'Relational Database',
+    url: '/certification/certifieduser/relational-database-v8'
+  },
+  {
+    name: 'Back End Development and APIs',
+    url: '/certification/certifieduser/back-end-development-and-apis'
+  },
+  {
+    name: 'Quality Assurance',
+    url: '/certification/certifieduser/quality-assurance-v7'
+  },
+  {
+    name: 'Scientific Computing with Python',
+    url: '/certification/certifieduser/scientific-computing-with-python-v7'
+  },
+  {
+    name: 'Data Analysis with Python',
+    url: '/certification/certifieduser/data-analysis-with-python-v7'
+  },
+  {
+    name: 'Information Security',
+    url: '/certification/certifieduser/information-security-v7'
+  },
+  {
+    name: 'Machine Learning with Python',
+    url: '/certification/certifieduser/machine-learning-with-python-v7'
+  },
+  {
+    name: 'College Algebra with Python',
+    url: '/certification/certifieduser/college-algebra-with-python-v8'
+  },
+  {
+    name: 'Foundational C# with Microsoft',
+    url: '/certification/certifieduser/foundational-c-sharp-with-microsoft'
+  }
+];
+
+const legacyCerts = [
+  { name: 'Front End', url: '/certification/certifieduser/legacy-front-end' },
+  { name: 'Back End', url: '/certification/certifieduser/legacy-back-end' },
+  {
+    name: 'Data Visualization',
+    url: '/certification/certifieduser/legacy-data-visualization'
+  },
+  {
+    name: 'Information Security and Quality Assurance',
+    url: '/certification/certifieduser/information-security-and-quality-assurance'
+  },
+  { name: 'Full Stack', url: '/certification/certifieduser/full-stack' }
+];
+
+test.use({ storageState: 'playwright/.auth/certified-user.json' });
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('/certifieduser');
+
+  // The following line is required if you're running the test in local development
+  // await page.getByRole('button', { name: 'Preview custom 404 page' }).click();
+});
+
+test.describe('Profile component', () => {
+  test('renders the camper profile correctly', async ({ page }) => {
+    await expect(
+      page
+        .getByText('ProfileDefault AvatarAn avatar coding with a laptop')
```

Comment:
The `DefaultAvatar` component has no ARIA role. This is the locator suggested by Playwright's VS Code extension.